### PR TITLE
Fix camelCase for product title in quickstart

### PR DESCRIPTION
**Pull request by rexagod**
Use camelCase for "OpenShift Container Storage".

### CODE REVIEW

File: controllers/storagecluster/quickstart_constants.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   name: "getting-started-ocs"
 spec:
   version: 4.7
-  displayName: "Getting started with Openshift Container Storage"
+  displayName: "Getting started with OpenShift® Container Storage"
```
Comment (umangachapagain) [inline controllers/storagecluster/quickstart_constants.go:12]: We just use `OpenShift Container Storage` everywhere. Why is this different?

**Review by umangachapagain — CHANGES_REQUESTED** [review]
Commit message can be updated to explain the change that is done and the reason for doing it instead of just linking a Bugzilla.

**Review by umangachapagain — COMMENTED** [review]
@rexagod PR to master does not really require a bugzilla. We require it only for backports.

File: controllers/storagecluster/quickstart_constants.go (hunk 1 above)
Comment (rexagod) [reply controllers/storagecluster/quickstart_constants.go]: Fixed.

Comment (vbnrh) [reply controllers/storagecluster/quickstart_constants.go]: @umangachapagain probably we should keep the (R) for consistency across the quickstarts. The install tour has it in its displayName as UXD wanted it there (for reasons unknown to me) . Either we change it here or we'll have to remove it in console-operator

**Comment by agarwal-mudit** [comment]
/retitle Non-unified reference to OCS in quickstarts

**Comment by agarwal-mudit** [comment]
Have removed BZ number from the title else bot will mark the Bug in MODIFIED state once the fix is merged.
Please use the BZ number while backporting it to 4.7

**Comment by umangachapagain** [comment]
> Commit message can be updated to explain the change that is done and the reason for doing it instead of just linking a Bugzilla.

@rexagod Seems like you missed this. Rest of it looks good.

**Comment by rexagod** [comment]
My bad, I thought you were referring to the issue body (https://github.com/openshift/ocs-operator/pull/1089#issue-577642506) so I updated that instead. Fixing ASAP.

**Comment by rexagod** [comment]
@umangachapagain PTAL.

**Comment by jarrpa** [comment]
The BZ should also be removed form the commit message. Also, the title shouldn't be quite so vague for such a small change. Considering changing the commit message to something like `Fix camelCase for product title in quickstart`

**Review by jarrpa — APPROVED** [review]
/lgtm

**Comment by agarwal-mudit** [comment]
@rexagod Please cherry-pick to release-4.7 You can do that automatically by saying /cherry-pick release-4.7

**Pull request merged**